src/ui-kit/experimental/patterns/layout/components/expansion/expansion-panel.html

```diff
@@ -1,7 +1,7 @@
 <ng-content select="mat-expansion-panel-header, md-expansion-panel-header, sam-expansion-panel-header"></ng-content>
 <div [attr.aria-hidden]="!expanded" class="mat-expansion-panel-content"
      [@bodyExpansion]="_getExpandedState()" [id]="id">
-  <div class="mat-expansion-panel-body">
+  <div *ngIf="expanded" class="mat-expansion-panel-body">
```

Comment:
The div above this should be hiding the items with the aria-hidden. And this was changed recently for this? Why this additional change?

Reply:
@thomastighe - Even though we have attribute aria-hidden to hide the content, Jaws does not recognize it and reads the content inside the accordion.